x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/components/index.ts

```diff
@@ -7,3 +7,5 @@
 
 export { PackagePolicyInputPanel } from './package_policy_input_panel';
 export { PackagePolicyInputVarField } from './package_policy_input_var_field';
+export { DatastreamPipeline } from './datastream_pipelines';
+export { DatastreamMappings } from './datastream_mappings';
```

Comment:
Do we need to do anything else to export this for other plugins?

Reply:
This will need to be in the plugin public/index file